analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExtensionMethodShouldNotExtendObject.vb

```diff
@@ -46,4 +46,20 @@ Module Noncompliant
     Public Sub MultipleConbinedAttributes(obj As Object) ' Noncompliant
     End Sub
 
+    <System.Runtime.CompilerServices.ExtensionAttribute>
+    Public Sub FullName(Arg As Object)          ' Noncompliant
+    End Sub
+
+    <Runtime.CompilerServices.ExtensionAttribute>
+    Public Sub ImplicitSystem(Arg As Object)    ' Noncompliant
+    End Sub
+
+    <System.Runtime.CompilerServices.Extension>
+    Public Sub ImplicitAttribute(Arg As Object) ' Noncompliant
```

Comment:
Descriptive names! Good. That is not always the case in some older test case files.